### ISSUE TITLE
Group discovery/list

### ISSUE THREAD

**Issue by Reaxt** [Auros/VRAtlas.Web#9]
**Is your feature request related to a problem? Please describe.**
As a new user, I would like to easily find new groups that may host events I would like to attend.

**Describe the solution you'd like**
It would be very nice to have a listing page of groups, maybe let groups give themselves tags on what kind of events they host. This would streamline the process of finding and following interesting things, while also allowing a user to find a community on the site first, rather than be redirected to the site from a third party to be able to find groups.

**Additional context**
It may make sense to have this section linked in the nav header.

**Comment by Auros** [comment]
Good idea. I do want to sit on this for a while as I want to see *how* people use groups. Group tags shouldn't be too bad as they could operate similarly to event tags.
